TriblerGUI/utilities.py

```diff
@@ -204,3 +204,13 @@ def get_gui_setting(gui_settings, value, default, is_bool=False):
     if is_bool:
         val = val == True or val == 'true'
     return val
+
+
+def is_dir_writable(path):
+    try:
+        open(os.path.join(path, 'temp'), 'w')
```

Comment:
Why don't you use `os.access`? https://docs.python.org/2/library/os.html#files-and-directories